tfx/components/evaluator/component.py

```diff
@@ -142,6 +147,11 @@ def __init__(
       absl.logging.warning('feature_slicing_spec is deprecated, please use '
                            'eval_config instead.')
 
+    if not example_splits:
+      example_splits = ['eval']
+      absl.logging.info("Computing metrics on the 'eval' split when "
+                        "example_splits is not set.")
```

Comment:
The 'example_splits' parameter is not set, using 'eval' split.

Reply:
Is this changed? Another option is to replace 'when' with 'because'

Reply:
yes, this is changed, try refresh the code or page, you should be able to see the change